app/stac_api/utils.py

```diff
@@ -322,13 +322,20 @@ def geometry_from_bbox(bbox):
         ValueError, IndexError, GDALException
     '''
     list_bbox_values = bbox.split(',')
+    if len(list_bbox_values) != 4:
```

Comment:
Most likely I got something wrong, but shouldn't be lengths of >4 be allowed, e.g. in case of polygons?
POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10)) This would result in a length of 5, or not?

Reply:
this is just about bbox